DevByRico PR

DevByRico opened the pull request:
Please include your Netlify link here.

DevByRico committed:
1th Commit

DevByRico committed:
README.md

JennieDalgren commented:
@RicardoNicolau take a look at this PR, it looks like you missed a gitignore file, 5000+ files (node modules got pushed)? Also, no Netlify link added.

DevByRico committed:
Remove node_modules and add .gitignore

DevByRico committed:
Add .gitignore to ignore node_modules

DevByRico committed:
Add .gitignore to ignore node_modules

DevByRico committed:
Update .gitignore with common ignores

DevByRico commented:
Netlify link - https://happy-thoughts-rn.netlify.app/

DevByRico commented:
Hi! 🙌
I've now added the missing .gitignore file and removed the node_modules from the repo.
Also added the Netlify live link above.
Let me know if there's anything else I should update! 😊

DevByRico committed:
README.md

DevByRico committed:
README.md

HIPPIEKICK reviewed (changes requested):
Please update your Happy thoughts app to either use our new API link that I posted about or your own deployed API that you're creating in the backend sprint.

Our link: https://happy-thoughts-api-4ful.onrender.com/

DevByRico committed:
fix(api): switch to new happy-thoughts-api-4ful.onrender.com URL

DevByRico commented:
I’ve updated the API URL to the new endpoint and removed any leftover node_modules. Ready for another review! 👍
